app/views/boards/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Board nr: <%= @show_id %> </h1>
```

Comment:
uzytj tutaj instancji obiektu która powinnas znalezc w kontrolerze#show.
@board = Board.find(params[:id]) => to powinno byc w kontrolerze i tutaj na widoku robisz:
@board.id => wyciaga id
@board.title => wyciaga title